components/generic/PageHeader.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <header class="row">
+    <b-col
+      cols="12"
+      :lg="mediaUrl ? 9 : 12"
```

Comment:
Could you always set this to 9. The title and description should always stretch 9 columns.

Reply:
Sorry, I forgot to push my changes